### PR TITLE
implemented a few methods in None

### PR DESCRIPTION
**Pull request by ryanguill**

### CODE REVIEW

File: src/option.ts [hunk 1]
```diff
@@ -106,10 +106,10 @@ export interface OptT<T> {
    *
    * ```
    * const maybeOne = OptionT.some(1);
-   * const one = maybeOne.unwrapOr(() => 3); // one === 1
+   * const one = maybeOne.unwrapOrElse(() => 3); // one === 1
    *
    * const maybeTwo = OptionT.none();
-   * const two = maybeTwo.unwrapOr(() => 3); // two === 3
```
Comment (iwburns) [inline src/option.ts]: oops, good catch

File: src/option/none.ts [hunk 2]
```diff
@@ -15,11 +15,11 @@ export class NoneT<T> implements OptT<T> {
     return true;
   }
 
-  expect(message: string): T {
```
Comment (iwburns) [inline src/option/none.ts]: neat! didn't know this was a thing.

File: src/option/none.ts [hunk 3]
```diff
@@ -70,5 +70,5 @@ export class NoneT<T> implements OptT<T> {
  * @returns {OptT<T>}
  */
 export function getNone<T>(): OptT<T> {
-  return new NoneT();
```
Comment (iwburns) [inline src/option/none.ts]: not holding the review up, but I'm curious as to why you made this change.

Did TS complain about it?  Or is this a readability thing?

Comment (ryanguill) [reply src/option/none.ts]: I'm sorry, I didnt really mean to commit that - I was getting some errors when running tsc and that was helping - but then I realized that I was running an older version of typescript, updating it fixed the issue - I'll send another PR to remove it.

**Pull request merged**